dashboard/test/ui/runner.rb

```diff
@@ -232,6 +232,7 @@ def format_duration(total_seconds)
   arguments += " -t ~@chrome" if browser['browserName'] != 'chrome' && !$options.local
   arguments += " -t ~@no_safari" if browser['browserName'] == 'Safari'
   arguments += " -t ~@no_firefox" if browser['browserName'] == 'firefox'
+  arguments += " -t ~@no_mobile_safari" if browser['browserName'] == 'iphone'
```

Comment:
Seems weird that @no_safari wouldn't include @no_mobile_safari. I wonder if we should rename these somehow. Maybe @no_safari_desktop and @no_ios? Also, for the cases where you block on iphone (currently @no_mobile_safari) do you also need to be blocking for ipad?